packages/legend-query-builder/src/components/explorer/QueryBuilderMilestoningEditor.tsx

```diff
@@ -156,7 +137,12 @@ const BiTemporalMilestoningEditor = observer(
           </div>
           <MilestoningParameterEditor
             queryBuilderState={queryBuilderState}
```

Comment:
bit concernered here with the usage of `guaranteeNonNullable` within the component. 
Have we caught where the nullable values are possible. Doing this in the component causes a total wipeout of the UI which is very bad UX. 

An exampe is `AllVersionsInRangelMilestoningParametersEditor`. you notify if its invaid (without start/end date ) but still render the components with the guaranteeNonNullable.

Reply:
My bad i forgot to remove the invalid part from there. It's not revelant anymore. Once we render these editors all these milestoning dates are non-null